src/install.rs

```diff
@@ -418,7 +465,12 @@ fn clear_partition_table(dest: &mut File, table: &mut dyn PartTable) -> Result<(
         .chain_err(|| "seeking to start of disk")?;
     let zeroes: [u8; 1024 * 1024] = [0; 1024 * 1024];
     dest.write_all(&zeroes)
-        .chain_err(|| "clearing partition table")?;
+        .chain_err(|| "clearing primary partition table")?;
+    // and now the backup GPT
+    dest.seek(SeekFrom::End(-(zeroes.len() as i64)))
```

Comment:
I'm concerned about this one:
 * the size seems way larger than the GPT (and in general it depends on the sector-size IIRC)
 * the offset seems off, and we may end up destroying legit data there

Reply:
Good catch!  I picked 1 MiB because that's what we use at the start of the disk, but we have a right to clobber that data and not the data at the end of the disk.  I'll reduce it to one sector.